### ISSUE TITLE
LazyTripletBatcher configuration

### ISSUE THREAD

**Issue by shengzhang0222** [philipperemy/deep-speaker#85]
Hi, I have some question about LazyTripletBatcher configuration.
self.nb_per_speaker = 2
self.nb_speakers = 640
self.history_length = 4
self.history_every = 100 
I'm training deep-speaker model on my own dataset, maybe I need to know what these parameters mean.
Thanks.

**Comment by philipperemy** [comment]
@shengzhang0222 you don't need to change those parameters. Those are internal.

**Comment by philipperemy** [comment]
We use a history (buffer) of triplets to train/test our models. And every `history_every=100` batches, we re-generate the history (buffer). In your case, I'd say you just need to update the nb_speakers. `history_length=4` controls how big your buffer is. `nb_speakers` is just the number of speakers you want to select from all the speakers. `nb_per_speaker`  is just a constant and you don't need to care about it. Positive samples are made of 2 utterances of the same speaker and this 2 is equal to `nb_per_speaker` from what I remember.

So long story short, just update `nb_speakers`.
